members.yaml

```diff
@@ -195,9 +195,99 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/wwi16ama/api_spec/master/openapi.yaml#/components/schemas/Error'
+  '/members/{id}/changePassword':
+    put:
+      summary: Change the password of a specific member
+      operationId: updatePasswordAsUser
+      tags:
+        - Members
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The id of the member who wants to update his password
+          schema:
+            type: integer
+            format: int32
+      requestBody:
+        description: Old and new Password
+        required: true
+        content:
+          application/json:
+            schema:
+              properties:
+                password:
+                  $ref: '#/components/schemas/Password'
+                newPassword:
+                  $ref: '#/components/schemas/Password'
+      responses:
+        '204':
+          description: Expected response to a valid request
+        '400':
+          description: When the old Password is not correct
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/wwi16ama/api_spec/master/openapi.yaml#/components/schemas/Error'
+        '401':
+          description: If the userId from the Authentification Header and the request-parameter id don't match
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/wwi16ama/api_spec/master/openapi.yaml#/components/schemas/Error'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/wwi16ama/api_spec/master/openapi.yaml#/components/schemas/Error'
+  '/members/{id}/changePasswordAdmin':
```

Comment:
Since you approved your own PR by yourself Im rewriting my suggestions again
Small suggestion. How about '/members/{id}/changePasswordAsAdmin' for a more descriptive route name